server/config/config.go

```diff
@@ -80,7 +80,8 @@ func configureConfigFile(vp *viper.Viper) {
 var ErrConfigFileNotFound = errors.New("config file not found")
 
 func readConfigFile(vp *viper.Viper) error {
-	if confFile := vp.GetString("config"); confFile != "" {
+	confFile := vp.GetString("config")
```

Comment:
There is a really weird bug that sometimes when you don't specify a `tracetest.yaml` config file the server can't start because of a weird issue trying to parse an empty file or something like that.

I added this but I think the initial bug needs to be fixed either way @schoren @mathnogueira